src/interfaces/IVelodromePoolB.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+interface IVelodromePoolB {
```

Comment:
this interface / contract should not be named `VelodromePoolB`. 
A & B are just named that you can use in your test, but it's not an official name to name contract in the repo.

when looking at optimistic etherescan, we see that:
- [this contract](https://optimistic.etherscan.io/address/0xe899b1b1d88e0198c668bb5500a0a4eda12ae3c8#code) is named `Pool`
- and [this contract]() is named `CLPool`

so you can name corresponding adapters:
- `NovaAdapterVeloPool`
- `NovaAdapterVeloClPool` (--> it corresponds to `NovaAdapterVeloPool` in the current codebase)